### PR TITLE
first pull request to conda-forge

### PR DESCRIPTION
**Pull request by adbeda**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pygeohash/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - pygeohash
+
+about:
+  home: https://github.com/wdm0006/pygeohash
+  license: AGPL-3.0
+  license_family: AGPL
+  license_file: LICENSE.txt
+  summary: 'A module for decoding and encoding geohashes to and from latitude and longitude coordinates'
+
+  # The remaining entries in this section are optional, but recommended.
```
Comment (chrisburr) [inline recipes/pygeohash/meta.yaml]: This should be removed

File: recipes/pygeohash/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  description: |
+    Pygeohash is a Python module that provides functions for
+    decoding and encoding geohashes to and from latitude and
+    longitude coordinates, and doing basic calculations and
+    approximations with them.
+
+
+extra:
+  recipe-maintainers:
+    - adbeda
```
Comment (chrisburr) [inline recipes/pygeohash/meta.yaml]: You need an extra line here to make the linter happy

File: recipes/pygeohash/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  files:
+    - tests/test_pygeohash.py
```
Comment (chrisburr) [inline recipes/pygeohash/meta.yaml]: It looks like you’re not using this file when testing?

File: recipes/pygeohash/meta.yaml (hunk 1 above)
Comment (adbeda) [reply recipes/pygeohash/meta.yaml]: @chrisburr are you referring to the comment section? Will do that right away

File: recipes/pygeohash/meta.yaml (hunk 2 above)
Comment (adbeda) [reply recipes/pygeohash/meta.yaml]: Got it. I noticed that when I looked into why it failed.

File: recipes/pygeohash/meta.yaml (hunk 3 above)
Comment (adbeda) [reply recipes/pygeohash/meta.yaml]: It is using the file and I do noticed logged outputs, but can double check on that

Comment (chrisburr) [reply recipes/pygeohash/meta.yaml]: You probably need to add a `commands` section to run whichever testing framework you use (adding it as a test dependency if required).

File: recipes/pygeohash/meta.yaml (hunk 2 above)
Comment (adbeda) [reply recipes/pygeohash/meta.yaml]: @chrisburr, this is my first contribution, what's the next step on this? should I go ahead and create another pull request after fixing the errors ?

Comment (chrisburr) [reply recipes/pygeohash/meta.yaml]: No need for another PR, just push to this branch again and the tests will run again.

**Comment by adbeda** [comment]
Now there are two empty lines at the end of the recipe - will be sending another push in just a few sec.

**Comment by adbeda** [comment]
@chrisburr all is fixed.

File: recipes/pygeohash/meta.yaml (hunk 3 above)
Comment (chrisburr) [reply recipes/pygeohash/meta.yaml]: You're currently not using this file. You should either remove the `files` section of `test` or add:
```yaml
  requires:
    - nose
  commands:
    - nosetests
```

Comment (adbeda) [reply recipes/pygeohash/meta.yaml]: Got it! just fixed that

**Pull request merged**